### PR TITLE
First loop - move imports from head into body.

### PR DESCRIPTION
**Pull request by garlicnation**

### CODE REVIEW

File: src/test/test.ts [hunk 1]
```diff
@@ -135,181 +130,166 @@ suite('Path Resolver', function() {
...
+      `<html><head>
+      <link rel="import" href="my-element/polymer/polymer.html">
+      <link rel="stylesheet" href="my-element/zork/my-element.css">
+      </head><body><dom-module id="my-element" assetpath="my-element/zork/">
+      <template>
+      <style>:host { background-image: url("my-element/zork/background.svg"); }</style>
+      </template>
+      </dom-module>
+      <script>Polymer({is: "my-element"})</script></body></html>`
     ].join('\n');
```
Comment (usergenic) [inline src/test/test.ts]: Shouldn't this be more like:

```
    var expectedBase = `
      <html><head>
      <link rel="import" href="my-element/polymer/polymer.html">
      <link rel="stylesheet" href="my-element/zork/my-element.css">
      </head><body><dom-module id="my-element" assetpath="my-element/zork/">
      <template>
      <style>:host { background-image: url("my-element/zork/background.svg"); }</style>
      </template>
      </dom-module>
      <script>Polymer({is: "my-element"})</script></body></html>`

Comment (usergenic) [inline src/test/test.ts]: (also I don't know if we care about the extra leading space now but maybe our test could have a util function to trimline like function trimlines(string) { string.replace(/^\s+/gm, '').trim() }; that we use on these heredoc style templates to ensure the leading indent doesn't leak semantic whitespace (text content nodes etc) into the test fixtures.

**Comment by usergenic** [comment]
Meh- still feel like the rename of test -> testdata is unnecessary.  Other than that, LGTM.

**Comment by usergenic** [comment]
Do tests need to pass to merge for this branch?  (I want to be clear on process, since I'm 'the new guy').

**Comment by garlicnation** [comment]
@usergenic I'm merging this into v3, not master. I don't expect the tests to be passing for the first few PRs, since there's too much functionality to capture in one change.

**Comment by garlicnation** [comment]
@usergenic The rename was unnecessary. I vastly simplified the test entry by switching back to `test/`

**Pull request merged**